src/wizard/deploy.jl

```diff
@@ -260,3 +257,26 @@ function step7(state::WizardState)
         println(state.outs, "```")
     end
 end
+
+function step7(state::WizardState)
+    print(state.outs, "Your build script was:\n\n\t")
+    println(state.outs, replace(state.history, "\n" => "\n\t"))
+
+    printstyled(state.outs, "\t\t\t# Step 7: Deployment\n\n", bold=true)
+    _deploy(state)
+end
+
+function deploy(state::WizardState = load_last_wizard_state(; as_is = true))
+    if state.step == :done
+        println(state.outs, "This state has been already deployed")
+        return
+    end
+    if state.step in (:step1, :step2)
+        println(state.outs, "This state cannot be deployed")
+        return
+    end
+    q = "This is an incomplete state, are you sure you want to deploy it?"
+    if yn_prompt(state, q, :n) == :y
+        _deploy(state)
```

Comment:
I'm undecided whether to reset the saved state at this point (for my own record: this can be done with `BinaryBuilder.save_last_wizard_state(BinaryBuilder.WizardState())`) or not.  For the time being I went with not resetting, in case the user wanted only to print the partial script and try to resume it later.

Reply:
I think that's fine.